### PR TITLE
chore(cli): update config file schema to include test config

### PR DESCRIPTION
**Pull request by roj1512**
These configurations were recently added by #15079, but the schema wasn't updated.

### CODE REVIEW

File: cli/schemas/config-file.v1.json [hunk 1]
```diff
@@ -322,5 +322,30 @@
       },
       "additionalProperties": false
     }
+  },
+  "test": {
```
Comment (dsherret) [inline cli/schemas/config-file.v1.json:326]: We'll need to move this into the "properties" object

image

**Review by dsherret — CHANGES_REQUESTED** [review]
Thanks! I forgot about this. One small change.

File: cli/schemas/config-file.v1.json (hunk 1 above)
Comment (roj1512) [reply cli/schemas/config-file.v1.json:326]: my bad

**Review by dsherret — APPROVED** [review]
LGTM. Thanks for following up with this @roj1512!

**Pull request merged**